src/main/java/wedlog/address/ui/DietaryPanel.java

```diff
@@ -39,8 +41,9 @@ public DietaryPanel(Logic logic) {
 
     private ObservableList<String> generateList() {
         ObservableList<String> dietaryList = FXCollections.observableArrayList();
-        logic.getDietaryRequirementStatistics().getMap()
-                .forEach((k, v) -> dietaryList.add(k + ": " + v));
+        logic.getDietaryRequirementStatistics().getMap().entrySet().stream()
```

Comment:
let's also include a count for "regular" meals that tallies up the guests whose d/ fields are empty